src/index.css

```diff
@@ -1,3 +1,277 @@
+:root {
+  --main-acent-color: #0290ff;
+}
+
 body {
-  background-color: white;
+  padding: 0;
+  margin: 0;
+  background: #fafafa;
+  box-sizing: border-box;
+  font-family: 'Montserrat', 'Times New Roman', Times, serif;
+}
+
+.d_flex {
+  display: flex;
+}
+
+.navigation_container {
+  background: #fff;
+  border-bottom: 1px solid #e8e8e8;
+}
+
+.navigation {
+  display: flex;
```

Comment:
- [ ] In line with `CSS` best practices, keeping our `CSS` dry, it is not nice to use the same rule repetitively, please kindly use a class attribute instead.

Reply:
Done